Docs/source/pages/get-started.md

```diff
@@ -273,27 +273,43 @@ AltTester package contains AltDriver class used to connect to the instrumented a
 
     .. tab:: C#
 
-        AltTester-Driver for C# is available also as a nuget package. You can use the nuget package to write your tests in a separate tests project, independent of the Unity application.
-
-        Create a new test project
-
-        .. code-block:: console
-
-            mkdir <test-project-name>
-            cd <test-project-name>
-            dotnet new nunit
-
-        Install AltTester-Driver nuget package
-
-        .. code-block:: console
-
-            dotnet add package AltTester-Driver --version 1.8.2
-
-        Run your tests
-
-        .. code-block:: console
-
-            dotnet test
+        **Prerequisite:**
+            - `.NET <https://dotnet.microsoft.com/en-us/download>`_ installed.
+            - Instrumented build.
+            - The AltTester Desktop installed on your PC.
+         
+        **Connection:**
+            From version Pro ``1.0``, the AltTester Desktop must be running on your PC while the tests are running.
```

Comment:
I don't know it will be Pro 1.0. I don't think we decided what version will be. This should not stop the merging of the ticket but I just want it to brought it up.

Reply:
We discussed with @sandraaltom related to this version and then we agreed that we can use Pro 1.0, but I agree with you that the whole team should decide after a discussion. Thank you!